cmd/cmrel/cmd/gcb_stage.go

```diff
@@ -175,8 +197,18 @@ func runGCBStage(rootOpts *rootOptions, o *gcbStageOptions) error {
 			}
 		}
 	}
-	// Add the 'manifests' file to the release output
-	if err := appendArtifact(&artifacts, o.RepoPath, "cert-manager-manifests.tar.gz", "", ""); err != nil {
+
+	manifestPostProcessor := func(path string) error {
+		if o.SkipSigning {
+			log.Println("skipping signing cert-manager-manifests.tar.gz because skip-signing is true")
+			return nil
+		}
+
+		return sign.CertManagerManifests(ctx, o.SigningKMSKey, path)
+	}
```

Comment:
>  I'm guessing we need it because we need access to the artifactPath variable?
yep

maybe I'll add a comment explaining that. can rename the func if you think it's worth it